SETUP/generate_css_from_less.sh

```diff
@@ -40,11 +37,11 @@ function run_less {
     css_file="${less_file%.*}.css"
     if [ $CHECK -eq 0 ]; then
         echo "Generating $css_file"
-        lessc $less_file $css_file
+        ../node_modules/.bin/lessc $less_file $css_file
```

Comment:
This is going to prevent running the script from an arbitrary directory. Consider using `$STYLE_DIR/../node_modules/.bin/lessc`.

Reply:
let me take a look at what was going on with my path then.

Reply:
I think your overall "lets use the one in `node_modules` is the right one. The PATH update on line 25 prioritized the system version which is not ideal. I just think we need to use a more flexible way to specify the `node_modules` lessc.